src/Raven.Studio/typescript/models/resources/widgets/databaseDiskUsage.ts

```diff
@@ -12,6 +12,7 @@ class databaseDiskUsage implements databaseAndNodeAwareStats {
     
     hideDatabaseName: boolean;
     even: boolean = false;
+    isCommonItem: boolean = false;
```

Comment:
can we avoid adding this prop on every existing items?

Reply:
`isCommonItem `is added on the 4 'per database' panels
and since we need it in line 23 in `nodeTagColumn.ts` then don't think we can remove it

Reply:
done - removed `isCommonItem `- using `nodeTag` to identify instead